packages/manager/apps/pci-instances/src/pages/onboarding/Onboarding.page.tsx

```diff
@@ -0,0 +1,5 @@
+import { FC } from 'react';
+
+const Onboarding: FC = () => <h1>Onboarding</h1>;
```

Comment:
this doesn't seem to be doing a lot of things

Reply:
Basically, it's a simple page component only to handle navigation redirection in `Instances.page.tsx`.